Dockerfile

```diff
@@ -11,6 +11,7 @@ ENV OVPN_FILES="https://downloads.nordcdn.com/configs/archives/servers/ovpn.zip"
     USERNAME="" \
     PASSWORD="" \
     COUNTRY="" \
+    GROUP="" \
```

Comment:
I thinks it's better to rename it to `SERVER_TYPE`, what do you think? When I take a look at the Nordvpn page you will see this: https://nordvpn.com/servers/tools/. This means renaming all the vars :).

Reply:
I had been naming it off their APIs; however, from a user perspective, I like `SERVER_TYPE` better.